src/SDKs/Authorization/Authorization.Tests/Authorization.Tests.csproj

```diff
@@ -15,10 +15,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ResourceManager" Version="1.1.0-preview" />
```

Comment:
@darshanhs90 you want to use latest Resource Manager 1.6.0 to record all your tests.
You are using 1.1.0-preview, which is really old and should no longer be used to record any tests.